solidity/contracts/utility/interfaces/ITime.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+/*
+    Time implementing contract interface
+*/
+interface ITime {
+    /**
+     * @dev returns the current time
+     */
+    function time() external view returns (uint256);
```

Comment:
Ahhh, it can't be `internal`.
But why do we need an `interface` here?
Contract `Time` should be an inherited extension (kinda like `RegistryClient`), no?

Reply:
I'll wait for your PR

Reply:
Done